pkg/identity/types.go

```diff
@@ -26,6 +25,31 @@ func (si ServiceIdentity) String() string {
 	return string(si)
 }
 
+// GetSDSCSecretName returns a string key used as the name of Certificate in all SDS structs.
+func (si ServiceIdentity) GetSDSCSecretName() string {
+	// TODO(draychev): The cert names can be redone to move away from using "namespace/name" format [https://github.com/openservicemesh/osm/issues/2218]
+	// Currently this will be: "service-cert:default/bookbuyer"
+	return si.ToK8sServiceAccount().String()
+}
+
+// GetCertificateCommonName returns a certificate CommonName compliant with RFC-1123 (https://tools.ietf.org/html/rfc1123) DNS name.
+func (si ServiceIdentity) GetCertificateCommonName() certificate.CommonName {
+	return certificate.CommonName(si)
+}
```

Comment:
Same concern as above, although not as serious as SDS specific types into the identity package.

Reply:
I will add this to the comments.  The plan is to have this here very temporarily to help with migration.

Both `GetSDSCSecretName()` and `GetCertificateCommonName()` are very helpful with this change.

I am going to create a GitHub issue w/ a follow up to remove these.

Reply:
Here is the issue:  https://github.com/openservicemesh/osm/issues/3182
Also added comments here to remove these 2 functions.

I find it easier to keep these here for a few days. Then once we have moved over, we will find a better place for these.